### PR TITLE
Adding QA editor block

### LINKED ISSUE #42: Add a QA Guttenberg block
We'll need to create a custom block to render QAs in Guttenberg.

Example Story: https://theworld.org/stories/2022-03-28/west-remains-divided-sanctions-weapons-ukraine-former-amb-russia-says

### PR DESCRIPTION
**Pull request by brandonhundt**
Closes #42 

- Adds a QA editor block
- Installs the Genesis Block Editor plugin

Notes:
- I'm not sure this is how we want to approach this, but it works.
- We'd have to create this QA block on the live env, I can't find a way to export the block settings itself.

## To Review

- [ ] Checkout Branch.
- [ ] Run `npm run local` (or `npm run refresh` if you want a fresh database.)
- [ ] Go to http://the-world-wp.lndo.site/.

> ...then...

- [ ] [Go here](http://the-world-wp.lndo.site:8000/wp-admin/edit.php?post_type=genesis_custom_block)
- [ ] Add a block called `QA`
- [ ] Add a text field called `question`
- [ ] Add a textarea called `answer`
- [ ] Publish
- [ ] Go to any post, add a new paragraph, and insert a "QA block"
- [ ] Add content to the QA block
- [ ] Note, the styling should reflect that of a QA block. 
- [ ] Inspect styling to confirm proper markup (it should match the frontend output)
- [ ] Confirm in API output.

### CODE REVIEW

File: wp-content/themes/newspack-theme/blocks/block-qa.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="qa-wrap">
+<div class="qa-question"><?php block_field( 'question' ); ?></div>
+<div class="qa-answer"><?php block_field( 'answer' ); ?></div>
+</div>
```
Comment (brandonhundt) [inline wp-content/themes/newspack-theme/blocks/block-qa.php:4]: This is file to define the markup.

File: wp-content/themes/newspack-theme/blocks/block-qa.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.qa-wrap {
```
Comment (brandonhundt) [inline wp-content/themes/newspack-theme/blocks/block-qa.css:1]: I pulled these styles from the frontend (via the inspector)

**Comment by rpeterman-gp** [comment]
@brandonhundt Is this still a WIP?

**Comment by brandonhundt** [comment]
@rpeterman-gp I removed the WIP label. Feel free to vet and give your opinion.

**Pull request merged**